### ISSUE TITLE
Visible link to extension help

### ISSUE THREAD

**Issue by elelay** [gpodder/gpodder#1329]
Currently it's behind a right-click with no indication one should right-click in the first place.
We should find something better

**Comment by elelay** [comment]
See #1319 for an example

**Comment by auouymous** [comment]
I agree. I didn't know about it until a week or two ago when I attempted to wrap long descriptions and found the menu in the code. Maybe an "info" or "menu" icon on each entry could open the menu, that way all three menu items become discoverable.
